### ISSUE TITLE
Invalid version format (non-numeric data)

### ISSUE THREAD

**Issue by eserte** [Leont/syntax-infix-smartmatch#4]
While building the distribution on one of my smokers:
```
Output from '/usr/perl5.38.2/bin/perl Build.PL':

Odd number of elements in hash assignment at /usr/perl5.38.2/lib/site_perl/5.38.2/ExtUtils/Builder/Planner.pm line 82.
Invalid version format (non-numeric data) at /usr/perl5.38.2/lib/site_perl/5.38.2/ExtUtils/Builder/Planner.pm line 84.
```

**Comment by Leont** [comment]
Yeah I need to bump the Dist::Build dependency to something that pulls in a recent enough ExtUtils::Builder::Planner. Will do that for the next release.

**Comment by Leont** [comment]
Fixed in 0.007
